distribution/src/config/jvm.options

```diff
@@ -33,16 +33,18 @@
 ################################################################
 
 ## GC configuration
--XX:+UseConcMarkSweepGC
--XX:CMSInitiatingOccupancyFraction=75
--XX:+UseCMSInitiatingOccupancyOnly
+8-13:-XX:+UseConcMarkSweepGC
+8-13:-XX:CMSInitiatingOccupancyFraction=75
+8-13:-XX:+UseCMSInitiatingOccupancyOnly
 
-## G1GC Configuration; to use G1GC uncomment the lines below
-# -XX:-UseConcMarkSweepGC
-# -XX:-UseCMSInitiatingOccupancyOnly
-# -XX:+UseG1GC
-# -XX:G1ReservePercent=25
-# -XX:InitiatingHeapOccupancyPercent=30
+## G1GC Configuration
+# to use G1GC, uncomment the next two lines and update the version on the
```

Comment:
Given that the below lines are already versioned to any version that supports CMS, do we need the instructions to update the version? Also, the instructions are a bit confusing, as they refer to uncommenting 3 lines, but the ones to be uncommented are 2 lines, and the ones to update the version on are the already uncommented 3 lines.

Reply:
@rjernst The instructions say to uncomment the next two lines, and then change the version on the following three lines (the three lines that follow the two lines that were uncommented). The reason that the version would need to be updated is in case the user is running on JDK 13 or previous and wants to use G1 GC.